app/src/main/res/values/dimens.xml

```diff
@@ -165,8 +165,10 @@
   <dimen name="content_item_exploration_view_padding_top">12dp</dimen>
   <dimen name="content_item_question_split_view_margin_bottom">12dp</dimen>
   <dimen name="content_item_question_view_margin_bottom">12dp</dimen>
-  <dimen name="feedback_item_exploration_view_padding_bottom">12dp</dimen>
-  <dimen name="feedback_item_exploration_view_padding_top">12dp</dimen>
+  <dimen name="feedback_item_exploration_view_padding_bottom">0dp</dimen>
+  <dimen name="feedback_item_exploration_view_padding_top">0dp</dimen>
+  <dimen name="feedback_text_view_padding_top">13dp</dimen>
+  <dimen name="feedback_text_view_padding_bottom">13dp</dimen>
```

Comment:
@Mehul-Bisht shouldn't we have top and bottom to be 12dp here?
`  <dimen name="feedback_text_view_padding_top">13dp</dimen>`
 ` <dimen name="feedback_text_view_padding_bottom">13dp</dimen>`

Reply:
@ARJUPTA I had to make this to 13 dp to fix the issue
![padding](https://user-images.githubusercontent.com/65572088/106069050-c44e0b80-6127-11eb-91cd-8791a437f6c7.png)